efficientdet/efficientdet_arch.py

```diff
@@ -694,3 +694,34 @@ def efficientdet(features, model_name=None, config=None, **kwargs):
       *utils.num_params_flops()))
 
   return class_outputs, box_outputs
+
+
+def freeze_backbone(variables):
```

Comment:
It seems a little over complicated to define multiple functions, how about allowing user to define regexpr as I suggested in hparams.py?